### PR TITLE
Improves performance of calendar view

### LINKED ISSUE #1256: Performance Improvement in Table view
Hey, @geekyd @Princu7 This is not the right approach to set width and current width every time the page loads with JS.
I hope you guys understand the effects of this on the performance of the app.
Proposed approach with some reasoning:

Can we do this calculation on the server calculation of offset we already know the column width and we already know the number of columns?
After getting the offset as a handlebar variable, we can add it to HTML DOM element using CSS style.
Please comment your suggestions @geekyd @Princu7. :)

### PR DESCRIPTION
**Pull request by geekyd**
Fixes #1256 

Changes: Computes the width adjustment for calendar view while generation of the web app and removes the width adjustment code from schedule.hbs. 

Demo: 
https://open-event-web-appp.herokuapp.com/

@mariobehling @aayusharora @Princu7 Please review. Thanks

### CODE REVIEW

**Comment by aayusharora** [comment]
@geekyd For a quick review, Can you please provide the gh-pages version to test.

File: src/backend/fold.js [hunk 1]
```diff
@@ -91,6 +93,15 @@ function createTimeLine(startTime, endTime) {
   };
 }
 
+function checkWidth(columns) {
```
Comment (aayusharora) [inline src/backend/fold.js]: Nice !! :+1:

File: src/backend/templates/schedule.hbs [hunk 2]
```diff
@@ -526,37 +523,6 @@
             $('.calendar-view').addClass('hide');
           });
 
-          function adjustCalendar(element) {
```
Comment (aayusharora) [inline src/backend/templates/schedule.hbs]: :1st_place_medal: For Getting rid of this. 
Looks correct.

**Comment by aayusharora** [comment]
@geekyd The code is nice and solves a problem but I have to test a live version to get this Merged.
Please provide the live link.

**Comment by geekyd** [comment]
@aayusharora Here is the link to gh-pages
https://geekyd.github.io/eventApplication/

**Comment by mariobehling** [comment]
@geekyd Would you be interested to solve the same issue in the Open Event Orga Server? https://github.com/fossasia/open-event-orga-server/issues/3555

**Comment by geekyd** [comment]
@mariobehling Sure I will try to solve it for the orga-server.

**Pull request merged**